lib/server.js

```diff
@@ -96,8 +93,9 @@ exports.create = function create(createOptions) {
       // render the componentInstance
       html += React.renderToString(componentInstance);
 
-      // state injection
-      html +=  format(TEMPLATE, Config.client.markupId, JSON.stringify(data));
+      // state (script) injection
+      var script = format(TEMPLATE, Config.client.markupId, JSON.stringify(data));
+      html = html.replace('</head>', script + '</head>');
```

Comment:
i think if you change to `html = html.replace('</body>',  script + '</body>');` this PR is good to go.